### PR TITLE
refactor: remove duplicated v2 ws enqueuing

### PR DESCRIPTION
**Pull request by jyeshe**
- Specialize only the order of key blocks (before or after micro blocks+transactions)
- Skips enqueuing micro blocks when there is no subscription

### CODE REVIEW

File: lib/ae_mdw_web/websocket/broadcaster.ex [hunk 1]
```diff
@@ -278,7 +290,22 @@ defmodule AeMdwWeb.Websocket.Broadcaster do
     |> Enum.uniq()
   end
 
+  defp get_subscribed_versions(channel) do
+    Enum.reduce([:v1, :v2], [], fn version, acc ->
```
Comment (sborrazas) [inline lib/ae_mdw_web/websocket/broadcaster.ex:294]: Same here 
```elixir
Enum.filter([:v1, :v2], &Subscriptions.has_subscribers?(&1, channel))
```

File: scripts/do.sh [hunk 2]
```diff
@@ -19,6 +19,6 @@ case $1 in
     ;;
 
   "test")
-    elixir --sname $NAME -S mix test
+    MIX_ENV=test elixir --sname $NAME -S mix test
```
Comment (sborrazas) [inline scripts/do.sh:22]: Not part of this PR I guess

**Review by sborrazas — COMMENTED** [review]
Nice performance improvement for the broadcaster!

File: lib/ae_mdw_web/websocket/broadcaster.ex (hunk 1 above)
Comment (jyeshe) [reply lib/ae_mdw_web/websocket/broadcaster.ex]: done

File: scripts/do.sh (hunk 2 above)
Comment (jyeshe) [reply scripts/do.sh]: done

**Review by sborrazas — APPROVED** [review]
